### DISCUSSION TITLE
mac install PVGeo

### DISCUSSION THREAD

**Discussion by stuartE9** [pyvista/pyvista#2224] [Q&A]
I am trying to install the PVGeo plugin for ParaView on Mac OSX 11.6.2 using the instructions on the PVGeo.org Getting Started page.  As recommended I have set up a virtual environment in Anaconda with python=2.7.  I can then activate the environment successfully. I already have ParavView 5.9, so skip that step.  

The trouble starts when I invoke `pip install PVGeo `.  Eleven packages download, the last being imageio that complains there is no python >=3.5.  This seems odd since we specifically set up an environment with python 2.7.  (The imageio web site is clear that it runs only on python >=3.5. )

So the install process halts and that's that.  My question is -- can this be fixed, and if so how?  Many thanks for any advice or workarounds.

**Comment by banesullivan** [comment]
Those instructions are quite outdated as ParaView has been updated to Python 3. I will see if I can update the instructions.

It's been really tough (for me) to maintain the ParaView support for PVGeo. If you're comfortable using Python, the preferred usage of PVGeo these days is with PyVista

**Comment by stuartE9** [comment]
thanks for the prompt response....  Look forward to any news on the revised instructions when  you have a chance.

Stu
